src/main/java/core/basesyntax/data/Main.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.data;
+
+import core.basesyntax.data.dao.FileReader;
+import core.basesyntax.data.dao.FileReaderImpl;
+import core.basesyntax.data.dao.FileWriter;
+import core.basesyntax.data.dao.FileWriterImpl;
+import core.basesyntax.data.model.FruitTransaction;
+import core.basesyntax.data.model.OperationType;
+import core.basesyntax.data.servise.FruitShopService;
+import core.basesyntax.data.strategy.BalanceOperationHandler;
+import core.basesyntax.data.strategy.OperationHandler;
+import core.basesyntax.data.strategy.PurchaseOperationHandler;
+import core.basesyntax.data.strategy.ReturnOperationHandler;
+import core.basesyntax.data.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String SOURCE_PATH_NAME = "src/data/fruit_shop_transactions";
+    private static final String TARGET_PATH_NAME = "src/reports/report";
```

Comment:
According to the checklist, input and output files should be placed in the `src/main/resources` folder. Consider updating the file paths to reflect this requirement.